cli/command/stack/swarm/common.go

```diff
@@ -48,3 +51,35 @@ func getStackSecrets(ctx context.Context, apiclient client.APIClient, namespace
 func getStackConfigs(ctx context.Context, apiclient client.APIClient, namespace string) ([]swarm.Config, error) {
 	return apiclient.ConfigList(ctx, types.ConfigListOptions{Filters: getStackFilter(namespace)})
 }
+
+// validateStackName checks if the provided string is a valid stack name (namespace).
+//
+// It currently only does a rudimentary check if the string is empty, or consists
+// of only whitespace and quoting characters.
+func validateStackName(namespace string) error {
+	v := strings.TrimFunc(namespace, quotesOrWhitespace)
+	if len(v) == 0 {
+		return fmt.Errorf("invalid stack name: %q", namespace)
+	}
+	return nil
+}
+
+func validateStackNames(namespaces []string) error {
+	for _, ns := range namespaces {
+		if err := validateStackName(ns); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func quotesOrWhitespace(r rune) bool {
+	if unicode.IsSpace(r) {
+		return true
+	}
+	switch r {
+	case '"', '\'':
+		return true
+	}
+	return false
```

Comment:
nit: wouldn't this be more readable
```
    return unicode.IsSpace(r) || r == '"' || r == '\''
```
?

Reply:
(which might optionally then be inlined into `validateStackName`)

Reply:
Good point; I initially had more stuff in the switch, but now it's not really needed anymore. I'll update